tern/formats/default/generator.py

```diff
@@ -101,7 +102,7 @@ def get_layer_info_list(layer):
     layer_file_licenses_list = []
     file_level_licenses = None
     package_list = PrettyTable()
-    package_list.field_names = ["Package", "Version", "License", "Pkg Format"]
+    package_list.field_names = ["Package", "Version", "License(s)", "Pkg Format"]
```

Comment:
small nit: This line is also now longer than 72 chars :( suggest breaking it up after `License(s)`